tsfile/src/main/resources/tsfile-format.properties.template

```diff
@@ -27,6 +27,9 @@ page_size_in_byte=1048576
 # The maximum number of data points in a page, defalut 1024*1024
 max_number_of_points_in_page=1048576
 
+# The acceptable error rate of bloom filter, should be in [0.01, 0.1]
+bloom_filter_error_rate=0.05
```

Comment:
I am afraid that just adding `bloom_filter_error_rate` to this `tsfile\src\main\resources\tsfile-format.properties.template` does not work.

Because if you check the start-server.bat/sh carefully, you will find that TSFILE_HOME equals IOTDB_HOME and IOTDB_HOME is default to be the direct parent directory of the start up script (so usually it will be like `...\server\target\iotdb-server-0.9.0-SNAPSHOT\`). That means in the TsFileDescriptor.loadProps function, the inputStream of properties will default to use the tsfile-format.properties under the `...\server\target\iotdb-server-0.9.0-SNAPSHOT\conf`, not this `tsfile\src\main\resources\tsfile-format.properties.template`. Note that `...\server\target\iotdb-server-0.9.0-SNAPSHOT\conf\tsfile-format.properties` is generated from `...\server\src\assembly\resources\conf\tsfile-format.properties`. 

So, of course this template should be updated, but `...\server\src\assembly\resources\conf\tsfile-format.properties` should also be updated.

Reply:
OK I will update that configure file